### ISSUE TITLE
training data set

### ISSUE THREAD

**Issue by HaynesLi** [AnkurDeria/Obstacle-Avoiding-AI-Car-Mlagents#1]
I followed the instruction of the project.
However I met the problem during the colab part in which I can not find the training data folder. 
Could you offer it? Thank!

**Comment by AnkurDeria** [comment]
There is no training data. When I mentioned path to the training folder I meant the directory where you want your generated models to be saved. The training data(input to the agent) is given by the unity simulation itself.

**Comment by HaynesLi** [comment]
Thus it should be a self-learning type?

**Comment by AnkurDeria** [comment]
Yes, you just reward the agent based on its output and the agent learns by trying to maximize the reward.

**Comment by HaynesLi** [comment]
But after I reviewed all codes in this project. I did not find where you define the network structure.

**Comment by AnkurDeria** [comment]
You only need to define the environment and the reward system. The network structure is predefined in ML-Agents.

**Comment by HaynesLi** [comment]
In which part the code using the ML-Agents api? I found the Addreward() function in your code.
